api/database/migrations/2024_04_10_200445_combine-assessment-result-note-fields.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        DB::statement("UPDATE assessment_results SET skill_decision_notes = CONCAT(
```

Comment:
I'm guessing most of the time not all these fields are filled and this will create a lot of unnecessary junk in the skill_decision_notes field.  I think it would be nice to only add these titles in if there's actually something in the other fields.  I'd be happy to help with the SQL if you want to try this.

Reply:
Nice catch! Okay I separated the statements and added a where clause to check if it's not null. If you have a better solution I'm happy to try that as well. Fixed by [6c5e28f](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10083/commits/6c5e28fcf5f5c62f59ea6c2ea2e0e19881a76a3e)

Reply:
Nice, thanks.  There's a small annoyance where if skill_decision_notes starts empty it gets an unnecessary newline when extra data is concatted in but I'm not too worried about that.